modules/account-baseline/modules/baseline-guardduty/cloudwatch.tf

```diff
@@ -0,0 +1,31 @@
+data "template_file" "cloudwatch_event_rule" {
+  template = <<TEMPLATE
+{
+  "source": [
+    "aws.guardduty"
+  ],
+  "detail-type": [
+    "GuardDuty Finding"
+  ]
+}
+TEMPLATE
+}
+
+
+resource "aws_cloudwatch_log_group" "this" {
+  name = local.guardduty_log_group_name
+  tags = var.tags
+}
+
+
+resource "aws_cloudwatch_event_rule" "this" {
+  name          = local.guardduty_event_rule_name
+  event_pattern = data.template_file.cloudwatch_event_rule.rendered
+}
+
+
+resource "aws_cloudwatch_event_target" "this" {
+  target_id = local.guardduty_event_target_id
+  rule      = aws_cloudwatch_event_rule.this.name
+  arn       = aws_cloudwatch_log_group.this.arn
+}
```

Comment:
missing EOF

Reply:
We don't have EditorConfig setup on this repo

Reply:
We do now https://github.com/ministryofjustice/hmpps-ems-platform-terraform-modules/pull/10/commits/29a0eabf591b4827e259132716e32736416a2059